classes/UserReputation.php

```diff
@@ -118,33 +118,33 @@ class UserReputation {
         Mission::RANK_S => 5
     ];
     const SPECIAL_MISSION_REP_GAINS = [
-        SpecialMission::DIFFICULTY_EASY => 1,
-        SpecialMission::DIFFICULTY_NORMAL => 2,
-        SpecialMission::DIFFICULTY_HARD => 3,
-        SpecialMission::DIFFICULTY_NIGHTMARE => 4,
+        SpecialMission::DIFFICULTY_EASY => 2,
+        SpecialMission::DIFFICULTY_NORMAL => 4,
+        SpecialMission::DIFFICULTY_HARD => 6,
+        SpecialMission::DIFFICULTY_NIGHTMARE => 8,
     ];
 
     const DAILY_TASK_REWARDS = [
         DailyTask::DIFFICULTY_EASY => [
             DailyTask::ACTIVITY_EARN_MONEY => 1,
-            DailyTask::ACTIVITY_ARENA => 10,
-            DailyTask::ACTIVITY_TRAINING => 4,
-            DailyTask::ACTIVITY_MISSIONS => 10,
-            DailyTask::ACTIVITY_PVP => 5,
+            DailyTask::ACTIVITY_ARENA => 20,
+            DailyTask::ACTIVITY_TRAINING => 10,
+            DailyTask::ACTIVITY_MISSIONS => 20,
+            DailyTask::ACTIVITY_PVP => 10,
         ],
         DailyTask::DIFFICULTY_MEDIUM => [
             DailyTask::ACTIVITY_EARN_MONEY => 2,
-            DailyTask::ACTIVITY_ARENA => 15,
-            DailyTask::ACTIVITY_TRAINING => 6,
-            DailyTask::ACTIVITY_MISSIONS => 15,
-            DailyTask::ACTIVITY_PVP => 10,
+            DailyTask::ACTIVITY_ARENA => 30,
+            DailyTask::ACTIVITY_TRAINING => 15,
+            DailyTask::ACTIVITY_MISSIONS => 30,
+            DailyTask::ACTIVITY_PVP => 15,
         ],
         DailyTask::DIFFICULTY_HARD => [
             DailyTask::ACTIVITY_EARN_MONEY => 3,
-            DailyTask::ACTIVITY_ARENA => 20,
-            DailyTask::ACTIVITY_TRAINING => 8,
-            DailyTask::ACTIVITY_MISSIONS => 20,
-            DailyTask::ACTIVITY_PVP => 15,
+            DailyTask::ACTIVITY_ARENA => 40,
+            DailyTask::ACTIVITY_TRAINING => 20,
+            DailyTask::ACTIVITY_MISSIONS => 40,
+            DailyTask::ACTIVITY_PVP => 20,
         ],
```

Comment:
I feel like grind/reward is pretty balanced here. Should we pump up money tasks to 5, 10, 15? It's strictly rank 1 content so not very abuseable.

Reply:
5/10/15 sounds good! Will make the change.